spec/support/wait_for_ajax.rb

```diff
@@ -0,0 +1,15 @@
+module WaitForAjax
```

Comment:
copied this from https://thoughtbot.com/blog/automatically-wait-for-ajax-with-capybara